src/components/menu-surface/menu-surface.scss

```diff
@@ -10,9 +10,14 @@
 @include menu.core-styles;
 
 .mdc-menu-surface {
-    width: var(--menu-surface-width, auto);
     max-height: 100%;
     position: relative;
+    --mdc-menu-max-width: var(
```

Comment:
I was thinking maybe we should mark this commit as a breaking change. This is probably no problem at all, but there might be someone who have had huge menus (due to long names on menu items) and they were happy with the size because their users could read the text.

Now after this their menu will render smaller, and grow max to 20 rem. (unless they override it). 

I don't know how much of a big deal this is. But it wouldn't hurt to add this as a new paragraph to the commit message:

````md
BREAKING CHANGE: menus' dropdown lists will not render wider than `20rem`, and their text will be truncated. If you have menus in which the whole menu item label has to be displayed for the end user, make sure to increase the menu width and read our docs to learn how to do it in a responsive way.
````

Reply:
Yeah, the breaking change might be a good idea. It feels like a case right on the line of what's usually considered breaking and not, but I think it's better to err on the side of caution. It's better that a change marked as breaking doesn't break anything for anyone, than that a change _not_ marked as breaking _does_ break something for someone 😅

Reply:
"If you have menus in which the whole menu item label has to be displayed for the end user"

We can't really guarantee that the whole item will be shown anyway, as this is just setting the max-width, right? So this is a little misleading. But Monday morning Adrian doesn't quite have the energy to come up with a better formulation. This is good _enough_, I think.